catroid/src/androidTest/java/org/catrobat/catroid/test/ProjectManagerTest.java

```diff
@@ -172,4 +174,46 @@ public void testLoadProjectWithInvalidNestingBrickReferences() throws IOExceptio
 
 		TestUtils.deleteProjects(PROJECT_NAME_NESTING_BRICKS);
 	}
+
+	@Test
+	public void testMergeProjectWithNoConflicts() throws IOException {
+		Project project1 = TestUtils.createProjectWithLanguageVersion(CURRENT_CATROBAT_LANGUAGE_VERSION,
```

Comment:
the createProjectWithLanguageVersion was intended to enable us to create projects with old language versions.

it also creates way more than what you need, so please create your own project here.